spec/requests/providers/has_other_proceedings_controller_spec.rb

```diff
@@ -136,11 +136,6 @@
 
         subject { delete providers_legal_aid_application_has_other_proceedings_path(legal_aid_application), params: params }
 
-        it 'sets a new lead application_proceeding_type when the original one is deleted' do
-          subject
-          expect(legal_aid_application.application_proceeding_types[0].lead_proceeding).to eq true
-        end
-
```

Comment:
Looks like this is from a different PR.

Reply:
it should have been in a previous PR yes, but was omitted, so I just put it in this one.